app/helpers/contact_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ContactHelper
+  def contact_link(contact)
+    if contact.present?
+      text = safe_join([fa_icon('envelope'), ' ', contact.name])
+      link_to(text, contact_path(contact))
+    end
+  end
+
+  def contact_from_constant(constant:)
```

Comment:
This is kind of smelly, but I prefer it to loading everything in views or having it all static.